apple/build_settings/build_settings.bzl

```diff
@@ -35,6 +35,12 @@ Declare a code signing identity, to be used in all code signing flows related to
     "use_tree_artifacts_outputs": struct(
         doc = """
 Enables Bazel's tree artifacts for Apple bundle rules (instead of archives).
+""",
+        default = False,
+    ),
+    "no_library_evolution": struct(
+        doc = """
+Disables library evolution and prevents the emission of .swiftinterface files.
```

Comment:
Could this be rephrased in a positive way, so is more natural to read/understand:

```
        default = True,
    ),
    "enable_library_evolution": struct(
```

Reply:
Depends on the conventions used in the rules. I didn't finy any flag like this enabled by default. As a alternative, we could go with `allow_internal_distribution` which nicely matches parameter and behavior from `xcodebuild`.

Reply:
I like `use_library_evolution`. We shouldn't have `no` in the name of a build setting, because then we get stuff like `--nono_library_evolution`.